pyvista/plotting/cube_axes_actor.py

```diff
@@ -289,12 +291,34 @@ def label_offset(self, offset: float):
         self.SetLabelOffset(offset)
 
     @property
-    def title_offset(self) -> float:  # numpydoc ignore=RT01
+    def title_offset(self) -> float | tuple[float, float]:  # numpydoc ignore=RT01
         """Return or set the distance between title and labels."""
+        if pyvista.vtk_version_info >= (9, 3):
+            offx, offy = (_vtk.reference(0.0), _vtk.reference(0.0))
+            self.GetTitleOffset(offx, offy)  # type: ignore[call-overload]
+            return offx, offy  # type: ignore[return-value]
+
         return self.GetTitleOffset()
 
     @title_offset.setter
-    def title_offset(self, offset: float):
+    def title_offset(self, offset: float | MutableSequence[float]):
+        vtk_geq_9_3 = pyvista.vtk_version_info >= (9, 3)
+
+        if vtk_geq_9_3:
+            if isinstance(offset, float):
+                msg = f'Setting title_offset with a float is deprecated from vtk >= 9.3. Accepts now a sequence of (x,y) offsets. Setting the x offset to {(x:=0.0)}'
+                warnings.warn(msg, UserWarning)
+                self.SetTitleOffset([x, offset])
+            else:
+                self.SetTitleOffset(offset)
+            return
+
+        if isinstance(offset, MutableSequence):
+            msg = f'Setting title_offset with a sequence is only supported from vtk >= 9.3. Considering only the second value (ie. y-offset) of {(y:=offset[1])}'
+            warnings.warn(msg, UserWarning)
+            self.SetTitleOffset(y)
+            return
+
```

Comment:
Due to vtk 9.3 deprecation [here](https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10072/diffs?file=829994b49e505c707ee9776d6359808d737edfad#829994b49e505c707ee9776d6359808d737edfad_172_177)